roles/openshift_health_checker/openshift_checks/mixins.py

```diff
@@ -2,17 +2,14 @@
 from openshift_checks import get_var
 
 
-class NotContainerized(object):
+class NotContainerizedMixIn(object):
```

Comment:
I'd suggest keeping mixins over metaclasses. While metaclasses are great for exposing apis in a easy to use form I feel like adding them where they are not extremely needed would be too magical and would likely cause confusion for those modifying the classes in the future.